activity.py

```diff
@@ -572,31 +583,30 @@ def add_value(self, label, value):
         except ValueError:
             _iter = self.model.append([label, str(value)])
 
-
         self.set_cursor(self.model.get_path(_iter),
                         self.get_column(1),
                         True)
 
-        _logger.info("Added: %s, Value: %s" % (label, value))
+        logging.debug("Added: %s, Value: %s" % (label, value))
```

Comment:
Puzzling that `_logger` is created, then instead of calling it you call the module.

Reply:
Done in 14f0084